### PR TITLE
Set the va-telephone component ESLint rule back to an error

### PR DESCRIPTION
**Pull request by jamigibbs**
## Description
Now that the [SMS option](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1246) has been added to the va-telephone component, we are setting the ESLint rule back to an error. The rule name change has been completed in https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/pull/888

## Original issue(s)
https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1257

## Testing done
VS Code with vets-website locally

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: .eslintrc.changed.js [hunk 1]
```diff
@@ -6,8 +6,8 @@ module.exports = {
     'jsx-a11y/anchor-is-valid': 2,
     'jsx-a11y/label-has-associated-control': 1,
     'jsx-a11y/no-static-element-interactions': 2,
-    '@department-of-veterans-affairs/prefer-telephone-component': 1,
-    '@department-of-veterans-affairs/telephone-contact-3-or-10-digits': 2,
+    '@department-of-veterans-affairs/prefer-telephone-component': 2,
+    '@department-of-veterans-affairs/telephone-contact-digits': 2,
```
Comment (bkjohnson) [inline .eslintrc.changed.js:10]: Have we updated the version of the ESLint plugin so that we get the changes?

Comment (jamigibbs) [reply .eslintrc.changed.js:10]: @bkjohnson I incremented to [v1.6.1](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/57de35a7e1f66cc1e29b55fd99ec7580dac04b27/packages/eslint-plugin/package.json#L3) did so I'm not sure why it's not picking it up. Published: https://www.npmjs.com/package/@department-of-veterans-affairs/eslint-plugin

Comment (jamigibbs) [reply .eslintrc.changed.js:10]: Maybe you mean in vets-website itself. I'll do that now but I thought it would pick up the minor bump.

Comment (bkjohnson) [reply .eslintrc.changed.js:10]: You'll probably need to do a `yarn install` to update the lock file.

**Review by bkjohnson — APPROVED** [review]

**Review by it-harrison — APPROVED** [review]

**Pull request merged**